src/test/java/org/folio/rest/impl/FundCodeUniquenessTest.java

```diff
@@ -0,0 +1,49 @@
+package org.folio.rest.impl;
+
+import io.restassured.response.Response;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+import java.net.MalformedURLException;
+
+import static org.folio.rest.utils.TestEntities.FUND;
+import static org.junit.Assert.fail;
+
+public class FundCodeUniquenessTest extends TestBase {
+
+  private final Logger logger = LoggerFactory.getLogger(FundCodeUniquenessTest.class);
+
+  @Test
+  public void testFundCodeUniqueness() throws MalformedURLException {
+
+    String sampleId = null;
+    try {
+      String fundSample = getFile("fund.sample");
```

Comment:
Please use any file from `src/main/resources/data/funds/` instead of `.sample` to work in unit tests with something real

Reply:
fixed unit test